src/main/java/com/subscribler/model/BusinessOwner.java

```diff
@@ -17,4 +19,6 @@
     private @NonNull String firstName;
     private @NonNull String lastName;
     private @NonNull String email;
+    private @NonNull List<Package> packageList;
+    private @NonNull List<Item> itemList;
 }
```

Comment:
/nit no new line here

Reply:
where should there be a new line?

Reply:
afaik, the convention is to put it in every final line of source code. I think it makes git's life easier when calculating the diffs

Reply:
fine... it was your code anyway